ocs_ci/framework/main.py

```diff
@@ -66,11 +70,66 @@ def init_ocsci_conf(arguments=None):
             os.path.expanduser(framework.config.RUN['bin_dir'])
         )
         utils.add_path_to_env_path(framework.config.RUN['bin_dir'])
+    print(framework.config)
     check_config_requirements()
 
 
+def sanitize_version_string(version):
```

Comment:
I don't think we need such function.  You can just use formatting in the place you need this value
on line 120: https://github.com/red-hat-storage/ocs-ci/pull/1281/files#diff-6e3f74e4d27052278e5be98b99e36c4eR120
conf_file_name = f"{version.replace('.', '_')}_default_config.yaml"

Reply:
done